easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2022.11-foss-2022.10.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2022.11'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'foss', 'version': '2022.10'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+builddependencies = [
+    ('hypothesis', '6.59.0'),
+    ('UnZip', '6.0'),
+    ('meson-python', '0.11.0'),
+    # installing numpy 1.23.5 from source requires setuptools < 60.0,
+    # cfr. https://github.com/numpy/numpy/issues/22157 and //github.com/numpy/numpy/issues/22431
+    ('setuptools', '63.4.3'),
```

Comment:
I deliberately didn't include the easyconfig for this in this PR, because we may want to favor downgrading the `setuptools` that comes with `Python-3.10.8-GCCcore-12.2.0.eb` (now that we still can since it's only in `develop`)

Reply:
Downgrading of `setuptools` included with `Python` is done in https://github.com/easybuilders/easybuild-easyconfigs/pull/16915

Reply:
`setuptools` build dependency removed in 1464932